From 15ad19b746fb8cf56c1f0ff1161b1f78a8771d82 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 03 Oct 2018 12:01:11 +0300
Subject: [PATCH] Makefile: goya-release: fix jenkins build start

The build has been parameterized and therefore should be accessed by
a different URL.
Fix the URL to support the parameterized build.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index d0d3093..51f9fa2 100644
--- a/Makefile
+++ b/Makefile
@@ -85,7 +85,7 @@
 	@-repo forall $(RELEASE_REPOS) -c git push origin :$(RELEASE_TAG_POINTER)
 	@repo forall $(RELEASE_REPOS) -c git push origin $(HL_FULL_VER) $(RELEASE_TAG_POINTER)
 	@echo "Released: $(HL_FULL_VER)"
-	@wget -q -O - http://192.168.0.4:8080/job/goya-release-test/build?token=goya-release
+	@wget -q -O - "http://192.168.0.4:8080/job/goya-release-test/buildWithParameters?token=goya-release"
 	@echo "Started the \"goya-release-test\" jenkins job... check your email for results."
 
 .PHONY: hl-goya hl-gaudi
